src/hooks/useActiveProject.tsx

```diff
@@ -50,35 +59,54 @@ export const ActiveProjectContextProvider = ({
         return;
       }
 
-      // TODO: Save for previously-selected project
+      storeProjectId(selectedProject.id);
```

Comment:
We need to `await` this, I'd think?

Reply:
We may want to catch any errors here but I'm not sure why we would need to wait for this to complete before proceeding. At this point, we've already set the activeAccountId in state so we really only need to store this for the next session. This goes to the below explanation about why I wrapped the get async storage with `react-query`, mostly to keep the effect cleaner without having to convert it to be async.

Reply:
But in this case, the callback is already `async`.  And agreed- if you still feel we shouldn't wait/verify it's stored before moving on, we need to (whether in this PR or down the road) log the error or something so it doesn't just go unnoticed.